docs/monitoring/docs/021-gs-monitoring-custom-metrics.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Expose Custom Metrics in Kyma
+type: Getting Started
+---
+
+## Overview
+
+This Getting started guide shows how to expose custom metrics to Prometheus with a Golang service in Kyma. To do so, follow these steps:
+
+1. Configure Istio
+2. Expose a sample application serving metrics on Port 8081.
+3. Access the exposed metrics in Prometheus
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- Istio 0.8
+  - Sidecar injection must be enabled ```istio-injection: "enabled"```
+  - Mutual TLS enabled.
+
+## Installation
+
+### Configure Istio
+
+Sidecar injection must be enabled, `istio-injection: "enabled"`
+for the `default` Namespace.
+
+To enable sidecar injection for all the pods running in the `default` Namespace, run the command below.
+
+```bash
+kubectl label namespace default istio-injection=enabled
+namespace "default" labeled
+```
+
+More details on deploying your application with Istio is [here](https://istio.io/docs/setup/kubernetes/quick-start#deploy-your-application) .
+
+It is needed to add the `sidecar.istio.io/inject` annotation with value `true` to the pod template spec to enable injection as done [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/deployment/deployment.yaml#L12).
+
+
+```yaml
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+```
+
+More details on installing the Istio Sidecar is here [here](https://istio.io/docs/setup/kubernetes/sidecar-injection/#policy).
+
+Following ports are used in the pod:
+1. Port: **8080**, as envoy will capture traffic only for ports listed in containerPorts of the pod, `containerPort: 8080`, or the list of ports in the annotation `traffic.sidecar.istio.io/includeInboundPorts`. Thus, this port is a part of the service mesh and can be used for application needs.
+
+2. Port: **8081**, the excluded port from service mesh and is used for exposing metrics only. So the network traffic **bypasses** Envoy and goes straight to the container. In Kyma, use the suggested port `8081` to expose metrics.
+
+### Expose a sample metrics application
+
+In order to expose prometheus metrics in Golang the prometheus community provides a library, [client_golang](https://github.com/prometheus/client_golang).
+
+This example has taken a basic example where `Gauge` and `Counter` metrics are exported using the package `prometheus`.
+
+1. Deploy the sample metrics application
```

Comment:
Dots are sad that you do not remember about them :( 
. . . . .

Reply:
done